packages/stylex/src/StyleXSheet.js

```diff
@@ -91,13 +91,22 @@ export class StyleXSheet {
     this.injected = false;
     this.ruleForPriority = new Map();
     this.rules = [];
+    this.constantsCache = new Map();
 
     this.rootTheme = opts.rootTheme;
     this.rootDarkTheme = opts.rootDarkTheme;
     this.supportsVariables =
       opts.supportsVariables ?? doesSupportCSSVariables();
   }
 
+  constantsCache: Map<
```

Comment:
nit: It's not a `cache`. It's just typical state. So maybe call this `constsMap`?